olive/passes/pytorch/genai_model_exporter.py

```diff
@@ -0,0 +1,91 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+# Export a PyTorch model using the onnxruntime-genai package.
+# --------------------------------------------------------------------------
+import logging
+from enum import Enum
+from pathlib import Path
+from typing import Any, Callable, Dict
+
+from onnx import TensorProto
+
+from olive.common.pydantic_v1 import validator
+from olive.hardware.accelerator import AcceleratorLookup, AcceleratorSpec, Device
+from olive.model import ONNXModelHandler, PyTorchModelHandler
+from olive.model.utils import resolve_onnx_path
+from olive.passes import Pass
+from olive.passes.olive_pass import PassConfigParam
+
+logger = logging.getLogger(__name__)
+
+
+class GenAIModelExporter(Pass):
```

Comment:
Since this generates an ONNX model, it should be under `olive.passes.onnx`

Reply:
+1

Reply:
Moved.